lms/templates/emails/activation_email.txt

```diff
@@ -1,22 +1,23 @@
 <%! from django.utils.translation import ugettext as _ %>
-<%! from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers %>
-${_("Thank you for creating an account with {platform_name}!").format(
-    platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
-)}
-
-${_("There's just one more step before you can enroll in a course: "
-    "you need to activate your {platform_name} account. To activate "
-    "your account, click the following link. If that doesn't work, "
-    "copy and paste the link into your browser's address bar.").format(
-        platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
-    )}
+${_("You're almost there!  Use the link to activate your account to access engaging, high-quality "
+"{platform_name} courses.  Note that you will not be able to log back into your account until "
+"you have activated it.").format(platform_name=platform_name)}
 
 % if is_secure:
   https://${ site }/activate/${ key }
 % else:
   http://${ site }/activate/${ key }
 % endif
-${_("If you didn't create an account, you don't need to do anything; you "
-      "won't receive any more email from us. If you need assistance, please "
-      "do not reply to this email message. Check the help section of the "
-      "{platform_name} website.").format(platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME))}
+
+${_("Enjoy learning with {platform_name}.").format(platform_name=platform_name)}
+
+${_("The {platform_name} Team").format(platform_name=platform_name)}
+
+${_("If you need help, please use our web form at {support_url} or email {support_email}.").format(
+  support_url=support_url, support_email=support_email
+)}
+
+${_("This email message was automatically sent by {lms_url} because someone attempted to create an "
+"account on {platform_name} using this email address.").format(
+  platform_name=platform_name, lms_url=lms_url
```

Comment:
Happy to do it, but why switch them?  I have them in the order they are being referenced?